babel.cfg

```diff
@@ -1,2 +1,2 @@
 [python: app.py]
-[jinja2: **/frontend/templates/**.html]
+[jinja2: **/frontend/magic_links/templates/**.html]
```

Comment:
I think we need a path that includes both i.e. `**/frontend/**.html` so we don't lose the existing templates.

Reply:
Aah you are right. I will add both paths to config